mods/default/mapgen.lua

```diff
@@ -782,6 +823,11 @@ elseif mg_params.mgname == "v6" then
 	default.register_ores()
 elseif mg_params.mgname == "v7" then
 	default.register_biomes()
+	default.register_decorations()
+	default.register_blobs()
+	default.register_ores()
+elseif mg_params.mgname == "singlenode" then
```

Comment:
Adding ores for singlenode mapgen has already been discussed : #445 (I was tired :-)
In fact, it's useless, because since these functions are public, the modder can use it.
It could even be a problem if ores and decorations are generated twice, or if the modder don't want ores.